sass/directives/_carousels.scss

```diff
@@ -105,7 +105,10 @@
   font-size: $direction-arrow-size / 2;
   cursor: pointer;
   border: 1px solid $inactive-carousel-button-color;
-  padding: $base-padding / 2;
+  text-align: center;
+  line-height: 28px;//this is for vertically centering arrows in ::after
+  height: 30px;
```

Comment:
Earlier I was using padding, but @arelia suggested this approach since the AC specifically calls for 30x30, if we use a variable like `$section-padding` then any changes to that variable will effect this element which might mean down the road there may be a ticket for making the arrow boxes back to 30.